### DISCUSSION TITLE
CrazyFlie 2.1 w. Mac M1

### DISCUSSION THREAD

**Discussion by esoasil** [bitcraze/discussions#89] [Q&A]
Dear All,

I'm trying to establish a connection between USB radio dongle with the Drone using Macbook with M1. The installation of prerequisites were successful. USB dongle is also recognized without problem. 

However if i tried to connect, the can't be found.

I run cfclient within pyenv 3.8.11  environment during start up following feedback can be seen in terminal
//-----
INFO:cfclient.gui:Disabling STL printouts
INFO:cfclient.gui:Foundation not found. Menu will show python as application name
INFO:cfclient.utils.input.inputreaders:Input readers: ['linuxjsdev', 'pysdl2']
INFO:cfclient.utils.input.inputreaders:Could not initialize [linuxjsdev]: Only supported on Linux
INFO:cfclient.utils.input.inputreaders:Successfully initialized [pysdl2]
INFO:cfclient.utils.input.inputinterfaces:Found interfaces: ['leapmotion', 'wiimote', 'zmqpull']
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [leapmotion]: Leap Motion library probably not installed (No module named 'leapsdk')
INFO:cfclient.utils.input.inputinterfaces:Could not initialize [wiimote]: Missing cwiid (wiimote) driver No module named 'cwiid'
------//

does anybody ever experienced the same problem?

Any hints would be very much appreciated.

Thanks a lot in advance

Best Regards
esoasil

**Comment by krichardsson** [comment]
Hi!

The log entries you list are probably not causing any problems as far as I can see, this is something else.

When you click the scan button in the Client, does the red LED on the Crazyradio light up?
Can you connect to the Crazyflie via USB?
